src/routes/safe/store/actions/fetchTokenBalances.js

```diff
@@ -5,10 +5,61 @@ import type { Dispatch as ReduxDispatch } from 'redux'
 
 import updateSafe from './updateSafe'
 
-import { getStandardTokenContract } from '~/logic/tokens/store/actions/fetchTokens'
+import { getOnlyBalanceToken, getStandardTokenContract } from '~/logic/tokens/store/actions/fetchTokens'
 import { type Token } from '~/logic/tokens/store/model/token'
 import { ETH_ADDRESS } from '~/logic/tokens/utils/tokenHelpers'
+import { sameAddress } from '~/logic/wallets/ethAddresses'
+import { ETHEREUM_NETWORK, getWeb3 } from '~/logic/wallets/getWeb3'
 import { type GlobalState } from '~/store/index'
+import { NETWORK } from '~/utils/constants'
+
+// List of all the non-standard ERC20 tokens
+const nonStandardERC20 = [
+  // DATAcoin
+  { network: ETHEREUM_NETWORK.RINKEBY, address: '0x0cf0ee63788a0849fe5297f3407f701e122cc023' },
+]
+
+// This is done due to an issues with DATAcoin contract in Rinkeby
+// https://rinkeby.etherscan.io/address/0x0cf0ee63788a0849fe5297f3407f701e122cc023#readContract
+// It doesn't have a `balanceOf` method implemented.
+const isStandardERC20 = (address: string): boolean => {
+  return !nonStandardERC20.find(token => sameAddress(address, token.address) && sameAddress(NETWORK, token.network))
+}
+
+const getBatchBalances = (tokens: List<Token>, safeAddress: string) => {
```

Comment:
I think it's not necessary to have `Batch` in function name, if we'd want to somehow mark that this particular function is fetching balances in batch, it's better to do that `get(Token)BalancesBatch`

Reply:
done!